ovs/lib/alba.py

```diff
@@ -266,8 +265,8 @@ def _add_backend_osds(alba_backend_guid, osds, domain, metadata):
         alba_backend = AlbaBackend(alba_backend_guid)
         config = Configuration.get_configuration_path(key=alba_backend.abm_cluster.config_location)
 
-        failures = []
-        unclaimed = []
+        failed_claims = []
+        unclaimed_osds = []
```

Comment:
Why rename this. I changed this a while ago to something more generic and now again its `unclaimed_OSDS` and `failed_claims` ? 
Either name them both using `osds` or none